deploy/roles/devtools6/templates/devtools-6.sh.j2

```diff
@@ -0,0 +1 @@
+source /opt/rh/devtoolset-6/enable
```

Comment:
Doesn't look like this template is actually used anywhere.  It might make sense to place it in /opt to make it easy to get to for users, and ensure the file mode is 755.